docs/guide/user-guide.md

```diff
@@ -1 +1,109 @@
 # User guide
+
+In the user guide you can find a description of all components belonging to 
+DeepESDL.
+
+There are three components of DeepESDL: 
+1. [DeepESDL JupyterLab](#deepesdl-jupyterlab)
+2. [DeepESDL xcube viewer](#deepesdl-xcube-viewer)
+3. [DeepESDL datacube resources](#deepesdl-datacube-resources)
```

Comment:
```suggestion
3. [DeepESDL datacubes](#deepesdl-datacube-resources)
```

To me _Datacube resources_  sounds like resources attached to or required for datacubes.